app/src/main/res/values-om-rET/strings.xml

```diff
@@ -254,6 +259,15 @@
 
   <!-- Onboarding -->
   <string name="onboarding_get_started">Jalqabi</string>
+  <string name="screenonboarding_intro_1">Barbaadi \&amp; argadhu \ndhukkubsattoota kumaatamoota \nwajjiin</string>
+  <string name="screenonboarding_intro_1_hypertension">Baayyina dhiiga</string>
+  <string name="screenonboarding_intro_2">Galmeessa \n tursiisi</string>
+  <string name="screenonboarding_intro_2_bp">Dhiibbaa dhiigaa</string>
+  <string name="screenonboarding_intro_2_and">\&amp;</string>
+  <string name="screenonboarding_intro_2_medicines">Dawaawwan</string>
```

Comment:
Can you do the same for all `screenonboarding_intro` strings just like we have in `strings.xml`